apps/src/templates/teacherDashboard/SelectSectionDropdown.jsx

```diff
@@ -0,0 +1,54 @@
+import React, {PropTypes} from 'react';
+import {connect} from 'react-redux';
+import i18n from '@cdo/locale';
+import {navigateToHref} from '@cdo/apps/utils';
+
+const styles = {
+  container: {
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'flex-end',
+    marginBottom: 10,
+  },
+  dropdown: {
+    marginLeft: 10,
+    marginBottom: 0,
+  },
+};
+
+class SelectSectionDropdown extends React.Component {
+  static propTypes = {
+    // Provided by redux.
+    sections: PropTypes.object,
+    selectedSectionId: PropTypes.number,
+  };
+
+  onChange = (event) => {
+    const sectionId = event.target.value;
+    navigateToHref(`/teacher_dashboard/sections/${sectionId}`);
```

Comment:
We should consider eliminating this navigation at some point. 100% agree with keeping it "for now" since this gives us parity with pegasus teacher-dashboard and lets us move off angular soonest, though.